backend/compact-connect/stacks/api_stack/v1_api/api_model.py

```diff
@@ -0,0 +1,494 @@
+# pylint: disable=protected-access
+# This class initializes the api models for the root api, which we then want to set as protected
```

Comment:
Perhaps being silly, but where is the use of a protected class here?

Reply:
In the model definitions, you can see we are setting/accessing protected member variables of the root `api` construct. Here is an example: 
```
 if hasattr(self.api, '_v1_query_providers_response_model'):
            return self.api._v1_query_providers_response_model
        self.api._v1_query_providers_response_model = self.api.add_model(
```

Notice here we are checking for the attribute `_v1_query_providers_response_model`, which is protected as denoted by the `_` prefix. If it doesn't exist we are then setting its value.

Reply:
Ah got it. I missed that. Thanks!